docs/source/nlp/bert_pretraining.rst

```diff
@@ -0,0 +1,104 @@
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+
+More details about parameters for offline data preprocessing:
+
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| **Parameter**                             | **Data Type**   | **Description**                                                                                              |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| train_ds.max_predictions_per_seq          | integer         | Maximum number of masked tokens in a sequence in the preprocessed data                                       |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+
+
+More details about parameters for online data preprocessing:
+
+
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| **Parameter**                             | **Data Type**   | **Description**                                                                                              |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| model.max_seq_length                      | integer         | The maximum total input sequence length after tokenization                                                   |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| model.mask_prob                           | float           | Probability of masking a token in the input text during data processing                                      |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| model.short_seq_prob                      | float           | Probability of having a sequence shorter than the maximum sequence length                                    |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+
+.. note::
+
+    For offline data preprocessing **model.tokenizer** is null.
+    For online data preprocessing **model.tokenizer** needs to be specified.
+
```

Comment:
maybe specify what tokenizer is going to be used and what tokenizer/vocab users would need when using this model on downstream tasks

How about adding a line here on how to use this pre-trained bert model + tokenizer for downstream tasks and maybe cross-reference in the https://github.com/NVIDIA/NeMo/blob/r1.0.0rc1/docs/source/nlp/nlp_model.rst

Reply:
done